src/core/globals.d.ts

```diff
@@ -1,10 +1,7 @@
-export {};
```

Comment:
FMI: how did you decide that this should live in src/core/globals.d.ts vs. src/core/window/globals.d.t.s?

Reply:
It was more an invalidation of my previous assessment. Initially, the thinking was "this is being added to Window, so lets put it in `#core/window`". But it turns out we have a ton of things added to the `Window` interface, and this testing property `karma` has nothing to do with `Window` or the functionality in `#core/window`. If we ever have `core/testing` it would belong in a `globals.d.ts` there